sources/dynamodb/streaming.go

```diff
@@ -253,9 +315,133 @@ func getRecords(streamClient dynamodbstreamsiface.DynamoDBStreamsAPI, shardItera
 	return result, nil
 }
 
-// ProcessRecord processes records retrieved from shards.
+// getColsAndSchemas provides information about columns and schema for a table.
+func getColsAndSchemas(conv *internal.Conv, srcTable string) (schema.Table, string, []string, ddl.CreateTable, error) {
+	srcSchema := conv.SrcSchema[srcTable]
+	spTable, err1 := internal.GetSpannerTable(conv, srcTable)
+	spCols, err2 := internal.GetSpannerCols(conv, srcTable, srcSchema.ColNames)
+	spSchema, ok := conv.SpSchema[spTable]
+	var err error
+	if err1 != nil || err2 != nil || !ok {
+		err = fmt.Errorf(fmt.Sprintf("err1=%s, err2=%s, ok=%t", err1, err2, ok))
+	}
+	return srcSchema, spTable, spCols, spSchema, err
+}
```

Comment:
This seems to be a generic method not specific to dynamodb so its better to place it in utility

Reply:
Okay! will do that.